omero/scientists/index.html

```diff
@@ -58,12 +58,12 @@ <h4>Analyze</h4>
                 </div>
             </div>
             <div class="column">
-                <a href="{{ site.baseurl }}/omero/features/organize/">
+                <a href="{{ site.baseurl }}/omero/features/share/">
```

Comment:
Was this just a typo in the previous version?

Reply:
the previous version was saying ``Share data with collaborators or your PI via URLs`` and indeed, it was pointing to the ``organize`` page. I imagine it was a case of not having enough resources to come up with a new icon and page just for this one feature.